### PR TITLE
Final Commit: Fix minor bug in winningest coach, update README, and a…

### PR DESCRIPTION
**Pull request by calforcal**
…dd functionality to Runner. Deleted unneeded file.

* **Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines
- [X] The branch created follows our guidelines
- [X] Tests for the changes have been added (for bug fixes/features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

This change will make a minor Fix to one of the methods, add functionality to the Runner file and delete an unneeded file.

* **What is the new behavior (if this is a feature change)?**

Fixes a bug with a return value in winningest coach and also adds some print statements to the Runner File.

* **Other information**:

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -291,14 +291,15 @@ def lowest_scoring_home_team
   end
 
   def winningest_coach(season)
-    season_str = season.to_s
+    season_str = season
```
Comment (jcjurado3) [inline lib/stat_tracker.rb:294]: Nice catch here!

File: lib/game_factory.rb
Comment (AlexKiselich) [inline lib/game_factory.rb:1]: Not needed, good catch

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -291,14 +291,15 @@ def lowest_scoring_home_team
...
     season_games = @game_teams.select { |game| game.game_id.to_s[0,4] == season_str[0,4] }
     games_by_coach = season_games.group_by(&:head_coach)
     win_percentages = {}
     games_by_coach.each do |coach, games|
       win_count = games.count { |game| game.result == "WIN" }
       loss_count = games.count { |game| game.result == "LOSS" }
-      total_games = win_count + loss_count
+      tie_count = games.count { |game| game.result == "TIE" }
+      total_games = win_count + loss_count + tie_count
```
Comment (jcjurado3) [inline lib/stat_tracker.rb:302]: This is what was throwing me off. I kept returning the wrong coach the first time and managed to pass the test for a different season. Glad we ran it through the spec harness to catch this!

File: runner.rb [hunk 3]
```diff
@@ -12,3 +12,12 @@
 
 stat_tracker = StatTracker.from_csv(locations)
 
+puts "Futbol Statistics:"
```
Comment (jcjurado3) [inline runner.rb:15]: Futbol!!!

File: runner.rb
Comment (AlexKiselich) [inline runner.rb:1]: Looks great

File: spec/stat_tracker_spec.rb [hunk 4]
```diff
@@ -346,7 +346,7 @@
         game_teams: game_teams_path
       }
       stat_tracker = StatTracker.from_csv(locations)
-      expect(stat_tracker.winningest_coach(20122013)).to eq("Claude Julien")
+      expect(stat_tracker.winningest_coach("20142015")).to eq("Alain Vigneault")
```
Comment (jcjurado3) [inline spec/stat_tracker_spec.rb:349]: Good catch!

File: README.md
Comment (AlexKiselich) [inline README.md:1]: Thanks for adding to the README

File: README.md [hunk 5]
```diff
@@ -1,3 +1,39 @@
...
+- ### Contributors
+  - Michael Callahan: [LinkedIn](https://www.linkedin.com/in/michaelcallahanjr/) && [GitHub](https://github.com/calforcal)
+  - Andy Weissman: [LinkedIn](https://www.linkedin.com/in/andy-weissman/) && [GitHub](https://github.com/andyweissman6)
+  - Jimmy Jurado: [LinkedIn](https://www.linkedin.com/in/jimmy-jurado-093568131/) && [GitHub](https://github.com/jcjurado3)
+  - Alex Kiselich: [LinkedIn](http://linkedin.com/in/alexanderkiselich) && [GitHub](https://github.com/AlexKiselich)
+
+
+  ### [Retro](https://futbol-project001.atlassian.net/wiki/spaces/SD/pages/131073/Futbol+PD+Group+Project)
+
+  ### [Figma](https://www.figma.com/community/file/1232352726292640230)
```
Comment (andyweissman6) [inline README.md:39]: these links are such a great idea! Whole README looks very clean and easy to read.

**Pull request merged**